src/test/java/com/luma/LumaTests.java

```diff
@@ -0,0 +1,307 @@
+package com.luma;
+
+import com.luma.beans.Product;
+import com.luma.components.*;
+import com.luma.pages.CartPage;
+import com.luma.pages.HomePage;
+import com.luma.pages.PDP;
+import com.luma.pages.PLP;
+import com.luma.utils.CommonUtils;
+import com.luma.utils.ServiceUtil;
+import jdk.jfr.Description;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.luma.utils.WaitUtils.*;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class LumaTests extends AbstractTest {
+
+    public WebDriver driver;
+    private final Logger logger = Logger.getLogger(LumaTests.class);
+
+    @BeforeMethod
+    public void setDriver() {
+        driver = getDriver();
+    }
+
+    @AfterMethod
+    public void closeDriver() {
+        driver.quit();
+    }
+
+    @Test
+    public void validateProductsOnPLPpage() {
```

Comment:
The term validate can imply an assertion is happening, for test methods, prefer:
verifyAddProductFromPLP